wazo_calld/plugins/config/http.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from flask import request
+from wazo_calld.http import AuthResource
+from xivo.auth_verifier import required_acl
+from jsonpatch import JsonPatch
+
+from .schemas import config_patch_schema
+
+
+class ConfigResource(AuthResource):
+    def __init__(self, config_service):
+        self._config_service = config_service
+
+    @required_acl('calld.config.read')
+    def get(self):
+        return self._config_service.get_config(), 200
```

Comment:
Should be restricted to master tenant only, otherwise it can leaks sensitive config